src/controllers/v1.controller.ts

```diff
@@ -258,14 +271,24 @@ export class V1Controller {
 
     try {
       const application = await this.fetchApplication(id, filter)
-      const { preferredNodeAddress, rpcID } = await this.checkClientStickiness(rawData)
 
       if (application?.id) {
+        const { stickiness = false, stickinessDuration = DEFAULT_STICKINESS_DURATION } = application
+
+        const stickinessData = stickiness
+          ? await this.checkClientStickiness(rawData)
+          : {
+              preferredNodeAddress: '',
+              rpcID: 0,
+            }
```

Comment:
same with the comment above (they should be the same object)

Reply:
If it is repeated code, yes, I agree.